### PR TITLE
[SUSHI-07] add files for sql injection detection

### PR DESCRIPTION
**Pull request by PhantHive**
Command line exec python backend for AI SQL prevention.

Need to do:

-> need to check for version compatibility as the AI model doesn't work with the current implementation (python 3.11 and TF.2 and keras version (package.json)

-> should check for command line vulnerability.

### CODE REVIEW

File: src/controllers/checkRequest.controller.ts
Comment (PhantHive) [reply src/controllers/checkRequest.controller.ts]: resolved in the last commit

**Comment by PhantHive** [comment]
Blocked by environment:
TF 2 version
Keras version
PY version 

Need to review the saved model.

**Comment by KreatorOf** [comment]
Waiting for @PhantHive review.

I pushed the requirements.txt.

Check the "checkSafety.controller.ts" to send res to BB.

Thanks,

**Comment by PhantHive** [comment]
Well, I guess it's time to merge this PR. Everything is ready, I will work on deploy process after this branch is merged.

PS: we didn't make any unit test for this controller. We have to expect bug on prod but this is no big deal :)

**Comment by PhantHive** [comment]
> Well, I guess it's time to merge this PR. Everything is ready, I will work on deploy process after this branch is merged.
> 
> PS: we didn't make any unit test for this controller. We have to expect bug on prod but this is no big deal :)

@PyyLog

**Review by KreatorOf — APPROVED** [review]
Everything is good for merge

**Pull request merged**
